dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsRowMapper.java

```diff
@@ -92,13 +93,13 @@
     public SubSnpCoreFields mapRow(ResultSet resultSet, int rowNum) throws SQLException {
         return new SubSnpCoreFields(
                 resultSet.getLong(SUBSNP_ID_COLUMN),
-                resultSet.getInt(SUBSNP_ORIENTATION_COLUMN),
+                Orientation.getOrientation(resultSet.getInt(SUBSNP_ORIENTATION_COLUMN)),
                 resultSet.getObject(REFSNP_ID_COLUMN, Long.class),
-                resultSet.getInt(SNP_ORIENTATION_COLUMN),
+                Orientation.getOrientation(resultSet.getInt(SNP_ORIENTATION_COLUMN)),
                 resultSet.getString(CONTIG_NAME_COLUMN),
                 resultSet.getLong(CONTIG_START_COLUMN),
                 resultSet.getLong(CONTIG_END_COLUMN),
-                resultSet.getInt(CONTIG_ORIENTATION_COLUMN),
+                Orientation.getOrientation(resultSet.getInt(CONTIG_ORIENTATION_COLUMN)),
                 LocusType.fromInteger(resultSet.getInt(LOC_TYPE_COLUMN)),
```

Comment:
It is a small detail, but using a common naming for `getOrientation` and `fromInteger` would be nice. I think the former is a bit more clear, as it states its purpose and can always be overloaded if necessary to use it with multiple data types.

Reply:
Yes, I also thought about that but didn't change it finally. I like more getOrientation, so I'm renaming fromInteger to getLocusType